src/services/infra/InfraService.ts

```diff
@@ -265,6 +265,11 @@ export default class InfraService {
     SiteLaunchCreateParams,
     AmplifyError | SiteLaunchError | MissingUserEmailError
   > {
+    const isAdmin = email.endsWith(".gov.sg")
+
+    if (!isAdmin) {
+      return errAsync(new SiteLaunchError("Not authorized to launch site"))
+    }
```

Comment:
Just checking, is there a reason why we don't attach the `checkIsSiteAdmin` middleware to this instead (so checking for site admin, rather than just checking the email)

Reply:
O.o Derp moment  
ae851f77f071e97bcfb6d1feb249b509de3f0f62